realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -242,7 +242,7 @@ static void setAuthServerImpl(AuthenticationServer authServerImpl) {
     }
 
     // Return the currently configured User store.
-    static UserStore getUserStore() {
+    public static UserStore getUserStore() {
```

Comment:
that was the tradeoff compared to adding a new public API `isLoggedOut` that might be confusing for the user with `isValid`. FWIW we do expose `setUserStore` as a public API as well